pages/common/jq.md

```diff
@@ -25,3 +25,7 @@
 - Output the value of a given key of each element in a JSON text from stdin:
 
 `cat {{file.json}} | jq 'map(.{{key_name}})'`
+
+- Output the value of a given key to a string:
```

Comment:
We should also explain the raw-output part.

Reply:
> With this option, if the filter’s result is a string then it will be written directly to standard output rather than being formatted as a JSON string with quotes. This can be useful for making jq filters talk to non-JSON-based systems.

https://stedolan.github.io/jq/manual/

Would something like a " (and disable JSON output)" suffix be enough?

Reply:
sure